daml_dit_if/api/__init__.py

```diff
@@ -38,6 +38,7 @@ class IntegrationResponse:
     """
 
     commands: Optional[Sequence[Command]] = field(default_factory=_empty_commands)
+    error_handler: Optional[Callable[[Exception], Awaitable[None]]] = None
```

Comment:
This may be a @da-tanabe question, but do we have any Dazl errors that cannot be fully captured in an `Exception`? The more I think about it, the more I think you have the correct signature here, but do want to confirm.

Reply:
@mschaef-da not really… if `submit` doesn't raise an exception and returns normally, you can pretty much assume that the command resulted in a successfully-sequenced transaction.